lib/decorators/auto-controller.decorator.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller } from '@nestjs/common';
+import { AUTO_CONTROLLER_WATERMARK } from '../interfaces';
+
+export function AutoController(): ClassDecorator {
```

Comment:
The controller needs an URL prefix address.
It doesn't seem to do anything at the moment.

This might be a good place to start.
https://github.com/nestjs/nest/blob/master/packages/common/decorators/core/controller.decorator.ts